manager/integration/tests/test_basic.py

```diff
@@ -3643,6 +3644,237 @@ def test_volume_toomanysnapshots_condition(client, core_api, volume_name):  # NO
                                                "status", "False")
 
 
+def prepare_data_volume_metafile(client, core_api, volume_name, csi_pv, pvc, pod, pod_make, data_path, test_writing_data=False, writing_data_path="/data/writing_data_file"):  # NOQA
+    """
+    Prepare volume and snapshot for volume metafile testing
+
+    Setup:
+
+    1. Create a pod using Longhorn volume
+    2. Write some data to the volume then get the md5sum
+    3. Create a snapshot
+    4. Delete the pod and wait for the volume detached
+    5. Pick up a replica on this host and get the replica data path
+    """
+    # create the pod with pv, pvc and write some data
+    test_pod_name, _, test_pvc_name, data_md5sum1 = \
+        prepare_pod_with_data_in_mb(
+            client, core_api, csi_pv, pvc, pod_make, volume_name,
+            data_path=data_path)
+
+    pod['metadata']['name'] = test_pod_name
+    pod['spec']['volumes'] = [{
+        'name': pod['spec']['containers'][0]['volumeMounts'][0]['name'],
+        'persistentVolumeClaim': {
+            'claimName': test_pvc_name,
+        },
+    }]
+
+    snap1 = create_snapshot(client, volume_name)
+
+    # choose the replica that is on this host
+    volume = client.by_id_volume(volume_name)
+    host_id = get_self_host_id()
+    for replica in volume.replicas:
+        if replica.hostId == host_id:
+            break
+
+    err_mgs = "Host-ID: {}, volume replica: {}".format(host_id, replica)
```

Comment:
```suggestion
    err_msg = "Host-ID: {}, volume replica: {}".format(host_id, replica)
```

You can also use f-string instead.
```
f'Host-ID: {host_id}, volume replica: {replica}'
```

Reply:
use ```f'Host-ID: {host_id}, volume replica: {replica}'``` instead. Thanks.